pcf/core/particle.py

```diff
@@ -44,6 +44,27 @@ class Particle(object, metaclass=MetaParticle):
     Type of particle
     """
 
+    # create logger
+    dirname = os.path.dirname(__file__)
+    dirname, x = os.path.split(dirname)
+    filename = os.path.join(dirname, 'logging/pcf.log')
+    logger = logging.getLogger(getpass.getuser())
+    logger.setLevel(logging.DEBUG)
+
+    # create console handler and set level to debug
+    ch = logging.FileHandler(filename)
+    ch.setLevel(logging.DEBUG)
+
+    # create formatter
+    formatter = jsonlogger.JsonFormatter(
+        '%(asctime)s - %(name)s - %(module)s - %(funcName)s - %(levelname)s - %(message)s')
```

Comment:
might want to have been json messaging. this would require changing the inputs to the info and debug calls. 

for example:
```logger.info"{0}: setting desired state to {1}".format(self.pcf_id, self.desired_state)```

to 
```logger.info"{msg:msg, pcf_id:id,state:state, desired_state:{}")```